apps/researcher/src/messages/nl/messages.json

```
@@ -11,6 +11,18 @@
     "communitiesDescription": "Elke gemeenschap kan haar eigen community starten op dit platform. Een community kan lijsten met objecten maken die verhalen vertellen over hun geschiedenis. Communities kunnen ook verhalen toevoegen aan de objecten in deze collectie.",
     "communitiesLink": "Hier zijn enkele communities om je te inspireren. Of <link>bekijk alle communities</link>."
   },
+  "Signup": {
+    "title": "Maak een account aan",
+    "WhyH2": "Waarom een account aanmaken?",
+    "WhyP1": "Een account geeft je toegang tot meer functionaliteiten in de website. Zo kun je bijvoorbeeld je eigen verhaal aan een object koppellen, gebruik maken van communities en lijsten maken om je objecten te beheren. De belangrijkste reden om een account aan te maken is dat we willen weten wie er data toevoegd. Hierdoor proberen anoniem spammen te voorkomen.",
+    "WhatH2": "Wat kan ik doen met een account?",
+    "AddH3": "Voeg je eigen verhaal aan een objects",
+    "AddP1": "Een van de belangrijkste functionaliteiten van deze site is het toegoegen van je eigen verhaal aan objecten. Zo kunnen gebruikers bijvoorbeeld vertellen hoe objecten in het verleden werden gebruikt of het object vertalen in hun eigen taal.",
```

Kleine aanpassingen voor grammatica en duidelijkheid.

```suggestion
    "AddP1": "Een van de belangrijkste functionaliteiten van deze site is het toevoegen van je eigen verhalen aan objecten. Gebruikers kunnen bijvoorbeeld vertellen hoe objecten gebruikt werden of het object benoemen in hun eigen taal.",
```